### PR TITLE
mysql: adjust all the user creation logic to fit 5.6 & 8.0

### PR DESCRIPTION
**Pull request by prettyshuang**

### CODE REVIEW

**Comment by BohuTANG** [comment]
Ping @Easontai for review.

File: src/raft/follower.go [hunk 1]
```diff
@@ -159,7 +159,9 @@ func (r *Follower) processHeartbeatRequest(req *model.RaftRPCRequest) *model.Raf
 			}
 
 			// before change master need check gtid, if local gtid bigger than remote gtid degrade to INVALID
-			r.degradeToInvalid(&gtid, &req.GTID)
+			if r.getMembers() > 2 {
```
Comment (TCeason) [inline src/raft/follower.go:162]: Is that normal when a two-node MySQL cluster's Leader exists local commit?

Comment (TCeason) [reply src/raft/follower.go:162]: Is it to avoid misjudgments that could lead to an abnormal MySQL cluster?

Comment (prettyshuang) [reply src/raft/follower.go:162]: If the total number of nodes is 2, semi-synchronous replication may degrade to asynchronous replication, in some cases, high availability is affected.

Comment (TCeason) [reply src/raft/follower.go:162]: > If the total number of nodes is 2, semi-synchronous replication may degrade to asynchronous replication, in some cases, high availability is affected.

Ok, I know. But only to a certain extent, and some of the risks. Is it right?

Comment (prettyshuang) [reply src/raft/follower.go:162]: yes

**Comment by TCeason** [comment]
LGTM

**Pull request merged**